.github/workflows/on_release.yml

```
@@ -25,9 +34,7 @@ jobs:
       GOOGLE_PLAY_APPLICATION_ID: app.futured.kmptemplate.android
       GOOGLE_PLAY_WHATSNEW_DIRECTORY: androidApp/whatsnew
       # TODO PROJECT-SETUP verify product flavor configuration
-      # Specifies API environment for KMP build.
-      # One of [dev|prod] as per configuration of Buildkonfig plugin in :shared:network:* Gradle module.
-      KMP_FLAVOR: 'prod'
+      KMP_FLAVOR: ${{ env.KMP_FLAVOR }}
```

actionlint (1.7.4)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)